src/routes/order.route.js

```diff
@@ -6,15 +6,15 @@ import {
 } from '../middleware';
 import {
   updateOrderStatus,
-  getOrderStatusEvents,
   getOrdersByUser,
+  getOrderByUser,
 } from '../controllers/order.controller';
 import { asyncWrapper } from '../helpers';
 
 const router = express.Router();
 
 router.get('/', isAuthenticated, getOrdersByUser);
-router.get('/sse', getOrderStatusEvents);
+router.get('/:orderId', getOrderByUser);
```

Comment:
if this controller throws error to the client , wrap your controller in `asyncWrapper`

Reply:
sure did

Reply:
sure did